### PR TITLE
COMMERCE-4433 Make CommerceContext extendable with custom CommerceContextFactory

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

File: modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/context/TestCommerceContext.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.commerce.test.util;
+package com.liferay.commerce.test.context;
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/context/TestCommerceContext.java:15]:
```suggestion
package com.liferay.commerce.test.util.context;
```

File: modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/context/TestCustomCommerceContext.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.commerce.test.util;
+package com.liferay.commerce.test.context;
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/context/TestCustomCommerceContext.java:15]:
```suggestion
package com.liferay.commerce.test.util.context;
```

File: modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/context/TestCustomCommerceContextFactory.java [hunk 3]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.commerce.test.util;
+package com.liferay.commerce.test.context;
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/context/TestCustomCommerceContextFactory.java:15]:
```suggestion
package com.liferay.commerce.test.util.context;
```

File: modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/context/TestCustomCommerceContextHttp.java [hunk 4]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.commerce.test.util;
+package com.liferay.commerce.test.context;
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/context/TestCustomCommerceContextHttp.java:15]:
```suggestion
package com.liferay.commerce.test.util.context;
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/context/TestCustomCommerceContextHttp.java (hunk 4 above)
Comment (riccardo-alberti) [reply modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/context/TestCustomCommerceContextHttp.java]: Done. Thanks

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**